content/md/cartridge-configuration/04-import-images-configuration.md

```diff
@@ -22,14 +34,44 @@ In the [connector configuration page](01-where-configuration.html), you need to
 | :-----------------------------| :---------------------: |
 | akeneoImageType               |  Images(images)        |
 
-
 # How can I retrieve images from the "Asset Collection" attributes?
 
 In the [connector configuration page](01-where-configuration.html), you need to select `Assets` for the following parameter to retrieve images from the "asset collection" attributes.
 
 | Connector parameter           | PIM information        |
 | :-----------------------------| :---------------------: |
 | akeneoImageType               |  Assets(assets)        |
+| akeneoAssetSystem             |  New Asset System (PIM version >= 4.0) (new) |
+|                               |  Old Asset System (PIM version < 4.0) (old)  |
+| akeneoImageImportType         |  Image Media File only (media_file)          |
+|                               |  Image Media Link only (media_link)          |
+|                               |  Both (both)                                 |
+
+With the Assets option selected (akeneoImageType):
+* `Asset System Version` (akeneoAssetSytem): Select what type of Asset system you use, for PAM (version < 4.0) select the old Asset system and for Asset Management system (version >= 4.0) select new asset system.
+* `Akeneo Image Import Type` (akeneoImageImportType): you can select here if you rather prefer to:
+** *import Image binaries only* (Image Media File only) In this mode image binaries will be linked to products.
+** *import Image media links only* (Image Media Links Only) In this mode image links will be linked to products using SFCC’s external Image Management system.
+** *import both media files and media links* (Both) In this mode image binaries will be linked to products and image links will be imported to product custom attribute with the ID matching the following pattern,
+```json
+akeneo_<camelizedAssetCode>_<camelizedAssetAttributeCode>
```

Comment:
"camelized"? What is it?

Reply:
Camel Case a way to write variables: https://en.wikipedia.org/wiki/Camel_case

This is maybe nerdy